engine/src/test/java/io/camunda/zeebe/engine/processing/streamprocessor/EngineErrorHandlingTest.java

```diff
@@ -40,7 +75,10 @@
 public final class EngineErrorHandlingTest {
 
   private static final String STREAM_NAME = "foo";
+  private static final ProcessInstanceRecord PROCESS_INSTANCE_RECORD = Records.processInstance(1);
```

Comment:
:wrench: I'd not define it here and just inline everywhere. It's not a complex statement, doesn't save much resources and actually inlining it makes some tests easier to follow. For example, I was confused about `shouldBlacklistInstance` until I realized that `PROCESS_INSTANCE_RECORD` is a process instance with key 1